recipe/meta.yaml

```diff
@@ -1,39 +1,42 @@
 {% set name = "fastcluster" %}
-{% set version = "1.1.26" %}
-{% set sha256 = "a202f44a3b06f5cf9cdba3c67d6c523288922d6e6a1cdf737292f93759aa82f7" %}
+{% set version = "1.2.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastcluster-{{ version }}.tar.gz
+  sha256: aab886efa7b6bba7ac124f4498153d053e5a08b822d2254926b7206cdf5a8aa6
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - python
-    - numpy >=1.9
+    - numpy {{ numpy }}
     - setuptools
     - wheel
+    - pip
   run:
     - python
     - {{ pin_compatible('numpy') }}
 
 test:
   imports:
     - fastcluster
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: http://danifold.net/fastcluster.html
+  home: https://danifold.net/fastcluster.html
   license: BSD-2-Clause AND GPL-2.0-or-later
```

Comment:
I think the license is only BSD-2-Clause, see https://github.com/dmuellner/fastcluster/blob/v1.2.6/README.txt#L24 and https://github.com/dmuellner/fastcluster/blob/v1.2.6/COPYING.txt

The license was changed in v1.1.8, see https://github.com/dmuellner/fastcluster/commit/ff8d0bff6a861ef60083f6a80bdba0504f39fda9#diff-95c20d744db732cdbca24c3e0406c10005ecf7fe8b5719c2fdf2b8af3fcedc79R114